src/Sylius/Behat/Context/Ui/Admin/ManagingShipmentsContext.php

```diff
@@ -34,11 +36,19 @@ final class ManagingShipmentsContext implements Context
     /** @var NotificationCheckerInterface */
     private $notificationChecker;
 
-    public function __construct(IndexPageInterface $indexPage, ShowPageInterface $orderShowPage, NotificationCheckerInterface $notificationChecker)
-    {
+    /** @var ShowPage */
+    private $shipmentShowPage;
```

Comment:
```suggestion
    private $showPage;
```
will be enough as it is a context for managing shipments